doc/api/dns.md

```diff
@@ -1036,24 +1116,45 @@ using `dnsPromises.lookup()`.
 
 Example usage:
 
-```js
+```mjs
+import dns from 'node:dns';
+const dnsPromises = dns.promises;
+const options = {
+  family: 6,
+  hints: dns.ADDRCONFIG | dns.V4MAPPED,
+};
+
+dnsPromises.lookup('example.org', options).then((result) => {
```

Comment:
```suggestion
await dnsPromises.lookup('example.org', options).then((result) => {
```

Reply:
Thank you for the review!

A quick question (I'm still learning :blush:), is it strictly necessary to `await` there? or is this change more of a stylistic change? Isn't the `then` already _waiting_ for the `result` to exist before executing the next portion of the code?

Thanks again! :raised_hands: I'll be adding your changes!

Reply:
It depends what you mean by “necessary”, but I’d argue it’s not a stylistic change, putting the `await` is about best practice. If you don’t await, and for some reason the promise never resolves, it’s going to be very hard to understand what went wrong – if you do await, the exit code will be 13 instead of 0, with a warning depending on what version you are using.
The `.then` is not waiting for the result, it’s registering a callback to call when/if the promise ever resolves.

Reply:
Thank you for taking the time to explain that to me :pray: ! Greatly appreciated!